Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
+      json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
-      rexml
-      rubocop-ast (>= 1.16.0, < 2.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.20.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.16.0)
+    rubocop-ast (1.21.0)
       parser (>= 3.1.1.0)
-    rubocop-performance (1.13.3)
+    rubocop-performance (1.14.3)
       rubocop (>= 1.7.0, < 2.0)
       rubocop-ast (>= 0.4.0)
     ruby-progressbar (1.11.0)
     rubyzip (2.3.2)
-    selenium-webdriver (4.1.0)
+    selenium-webdriver (4.4.0)
       childprocess (>= 0.5, < 5.0)
       rexml (~> 3.2, >= 3.2.5)
-      rubyzip (>= 1.2.2)
-    sidekiq (6.4.1)
-      connection_pool (>= 2.2.2)
+      rubyzip (>= 1.2.2, < 3.0)
+      websocket (~> 1.0)
+    sidekiq (6.5.6)
+      connection_pool (>= 2.2.5)
       rack (~> 2.0)
-      redis (>= 4.2.0)
+      redis (>= 4.5.0, < 5)
     simple_form (5.1.0)
       actionpack (>= 5.2)
       activemodel (>= 5.2)
-    sshkit (1.21.2)
+    sshkit (1.21.3)
       net-scp (>= 1.1.2)
       net-ssh (>= 2.8.0)
     sshkit-interactive (0.3.0)
       sshkit (~> 1.12)
-    standard (1.9.1)
-      rubocop (= 1.26.1)
-      rubocop-performance (= 1.13.3)
-    stimulus-rails (1.0.4)
+    standard (1.16.1)
+      rubocop (= 1.35.1)
+      rubocop-performance (= 1.14.3)
+    stimulus-rails (1.1.0)
       railties (>= 6.0.0)
-    strscan (3.0.1)
+    strscan (3.0.4)
     thor (1.2.1)
-    timeout (0.2.0)
-    turbo-rails (1.0.1)
+    timeout (0.3.0)
+    turbo-rails (1.1.1)
       actionpack (>= 6.0.0)
+      activejob (>= 6.0.0)
       railties (>= 6.0.0)
-    tzinfo (2.0.4)
+    tzinfo (2.0.5)
```

Comment:
### [tzinfo](https://tzinfo.github.io)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.0.4 → 2.0.5

[_[change-log](https://github.com/tzinfo/tzinfo/blob/master/CHANGES.md), [source-code](https://github.com/tzinfo/tzinfo/tree/v2.0.5), [gem-diff](https://my.diffend.io/gems/tzinfo/2.0.4/2.0.5)_]





<details>
<summary>Commits</summary>

A change of **30** commits. See the full changes on [the compare page](https://github.com/tzinfo/tzinfo/compare/v2.0.4...v2.0.5).

These are the first 10 commits:
- (c357395) [[ci skip] Improve formatting.](https://github.com/tzinfo/tzinfo/commit/c3573959f6740eed8794a47afc33a6615b0d6a29)
- (c91bcde) [[ci skip] Add version 0.3.59 from the 0.3 branch.](https://github.com/tzinfo/tzinfo/commit/c91bcde8d39f7d67283e2954145176c4e0e3a69d)
- (fdddd45) [Switch to GitHub Actions for CI.](https://github.com/tzinfo/tzinfo/commit/fdddd4583f343616fd1efb0747c0db5efe08b01f)
- (8037cd0) [Limit json to < 2.5.0 on Windows Ruby 2.0.](https://github.com/tzinfo/tzinfo/commit/8037cd0513a8f945801c14dbd06a225802480589)
- (43df865) [Mark truffleruby as experimental.](https://github.com/tzinfo/tzinfo/commit/43df8650944d957c459a58bbd4d566b79b7d4e67)
- (0cb6da2) [Ignore more warnings from sub-processes.](https://github.com/tzinfo/tzinfo/commit/0cb6da27b7827201e4c0657bacff4a54e3c57146)
- (f29fe6f) [Add RubyGems logo.](https://github.com/tzinfo/tzinfo/commit/f29fe6fc13aa949f6a68985898ddd2a7c3de4e11)
- (be39bfa) [Remove an unnecessary or.](https://github.com/tzinfo/tzinfo/commit/be39bfad643f379204b75e282c61c312fdf3fb40)
- (f4a8d0d) [Add version 0.3.60 from the 0.3 branch.](https://github.com/tzinfo/tzinfo/commit/f4a8d0d63d20c12d0889ebe6753ce5c9828c4914)
- (8d87a69) [Fix a grammatical error.](https://github.com/tzinfo/tzinfo/commit/8d87a69eaf39e8fcfbac5550003ffc8763b5c1dd)


</details>